test/simplify_algebra_test.cpp

```diff
@@ -236,6 +236,30 @@ TEST_CASE(simplify_mul_conv1)
     EXPECT(new_conv->outputs().front()->name() != "mul");
 }
 
+TEST_CASE(simplify_mul_conv2)
+{
+    migraphx::module m;
+    auto x = m.add_parameter("x", {migraphx::shape::int32_type, {1, 128, 28, 28}});
+    auto w =
+        m.add_literal(migraphx::generate_literal({migraphx::shape::int32_type, {256, 128, 3, 3}}));
+    auto conv = m.add_instruction(
+        migraphx::make_op("convolution",
+                          {{"padding", {1, 1}}, {"stride", {2, 2}}, {"dilation", {1, 1}}}),
+        x,
+        w);
+    auto a      = m.add_literal(migraphx::generate_literal({migraphx::shape::int32_type, {256}}));
+    auto unsq_a = m.add_instruction(migraphx::make_op("unsqueeze", {{"axes", {1, 2}}}), a);
```

Comment:
I think multibroadcast can do unsqueeze itself.

Reply:
We can keep this, I think it tests the `squeeze` line in the matcher.

Reply:
I tried this out when rewriting batch_norm, it doesn't work without the unsqueeze. And AFAIK if we follow the numpy broadcasting rules it would try to broadcast to the last dimension rather than the channels (2nd dimension).